api/src/app/modules/spatial-feature/spatial-feature.controller.ts

```diff
@@ -31,9 +32,15 @@ export class SpatialFeatureController {
       throw new BadRequestException('Invalid version');
     }
 
-    this.logger.debug('Start get /spatial-feature/bcgw-extract'); // For measuring performance.
-  
-    return this.spatialFeatureService.getBcgwExtract();
+    this.logger.info('Start get /spatial-feature/bcgw-extract'); // For measuring performance.
+
+    const start = performance.now();
+    const result = this.spatialFeatureService.getBcgwExtract();
+    const end = performance.now();
+
+    this.logger.info(`End get /spatial-feature/bcgw-extract for ${end - start}ms.`);
```

Comment:
We likely able to config the Logger to measure the performance for this particular class functions, but for now, just do the basic one inside the function.

Reply:
Doing the time calculation isn't really necessary (can calculate manually from logging timestamps), but doesn't hurt - might as well leave in.